src/solver/conv/conv_wino_fury_RxS.cpp

```diff
@@ -255,6 +255,15 @@ bool ConvWinoFuryRxSCommon<Winodata, Winofilter>::IsApplicable(const ExecutionCo
     if(!StartsWith(dev_name, "gfx11"))
         return false;
 
+    if(StartsWith(dev_name, "gfx115"))
+    {
+        // Triggers this error on gfx1151
+        // kernel: [drm:gfx_v11_0_bad_op_irq [amdgpu]] *ERROR* Illegal opcode in command stream
+        // stderr: HSA_STATUS_ERROR_INVALID_ISA: The instruction set architecture is invalid. code:
+        // 0x100f
+        return false;
+    }
```

Comment:
This should be explicitly marked as a workaround. Also this isn't needed for Windows.


```suggestion
#if WORKAROUND_ISSUE_3735
    if(dev_name == "gfx1151")
    {
        return false;
    }
#endif
```


At the beginning of this file, the following should be added:

```code
// There is a bug in the Linux driver for gfx1151: the VGPRs count check is incorrect.
#if __linux__
#define WORKAROUND_ISSUE_3735 1
#else
#define WORKAROUND_ISSUE_3735 0
#endif
```